docker/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+apk add --update go build-base git mercurial ca-certificates
```

Comment:
You probably want `--no-cache` instead of `--update`.. then you don't have to delete `/var/cache/apk/*` later.

Reply:
You can also add an option like `--virtual .build-deps` then later just do a `apk del .build-deps`.

OTOH, there may actually be no need to delete things after use as this is a builder image.

Reply:
`mercurial`?

Reply:
This file is copied verbatim from https://github.com/gliderlabs/logspout/blob/master/custom as per the instructions there.

Reply:
mercurial is needed by go get for non-git sources (I think)